doc/examples/applications/plot_cornea_spot_inpainting.py

```diff
@@ -0,0 +1,197 @@
+    height=500,
+    width=500,
+    binary_string=True,
+    labels={'animation_frame': 'time point'},
+    title='Sample of in-vivo human cornea'
+)
+plotly.io.show(fig)
+
+#####################################################################
+# Average over time
+# =================
+# First, we want to detect those dark spots where the data are lost. In
+# technical terms, we want to *segment* the dark spots. We want to do so for
+# all frames in the sequence. Unlike the actual data (signal), the dark spots
+# do not move from one frame to the next; they are still. Therefore, we begin
+# by computing the time average of the image sequence. We shall use this
+# time-averaged image to segment the dark spots, the latter then standing out
+# with respect to the background (blurred signal).
+
+image_avg = np.mean(image_seq, axis=0)
+
+print(f'shape: {image_avg.shape}')
+
+fig = px.imshow(
+    image_avg,
+    width=500,
+    height=500,
+    binary_string=True,
+    title='Time-averaged image'
+)
+plotly.io.show(fig)
+
+#####################################################################
+# Use local thresholding
+# ======================
+# To segment the dark spots, we use thresholding. The images we are working
+# with have uneven illumination, which causes variations in the (absolute)
+# intensities of the foreground and the background, from one region to another
+# (distant) one. It is therefore more fitting to compute different threshold
+# values across the image, one for each region. This is called adaptive (or
+# local) thresholding, as opposed to the usual thresholding procedure which
+# employs a single (global) threshold for all pixels in the image.
+#
+# Let us compare the visibility of the hidden data in two different
+# thresholding masks, one of which has the `block_size` set to 21, while
+# the other has it set to 35. For this, we start by defining a convenience
+# function to create a mask:
+
+def create_mask(image_avg, spot_size):
+    thresh_value = filters.threshold_local(
+        image_avg,
+        block_size=spot_size,
+        offset=10
+    )
+    mask = (image_avg > thresh_value)
+    return 1 - mask
+
+#####################################################################
+# Let's also define a function to display two plots side-by-side so
+# that it is easier for us to draw comparisons between plots:
+
+def plot_comparison(plot1, plot2, title1, title2):
+    fig, (ax1, ax2) = plt.subplots(
+        ncols=2,
+        figsize=(12, 8),
+        sharex=True,
+        sharey=True
+    )
+    ax1.imshow(plot1, cmap='gray')
+    ax1.set_title(title1)
+    ax1.axis('off')
+    ax2.imshow(plot2, cmap='gray')
+    ax2.set_title(title2)
+    ax2.axis('off')
+
+#####################################################################
+# Now, we plot the two masks. It seems that the dust-covered
+# spots appear more distinct in the second mask!
+
+mask1 = create_mask(image_avg, 21)
+mask2 = create_mask(image_avg, 35)
```

Comment:
```suggestion
mask1 = 1 - (image_seq_mean > threshold_value1)
mask2 = 1 - (image_seq_mean > threshold_value2)
```
Maybe we can add a comment why inverse the mask

Reply:
Great spot. 
What the `1 - (image_avg > thresh_n)` does is exactly as you have explained here. 
Your implement is better. I completely agree with this